src/main/java/at/hannibal2/skyhanni/features/misc/limbo/LimboPlaytime.kt

```diff
@@ -0,0 +1,111 @@
+package at.hannibal2.skyhanni.features.misc.limbo
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.InventoryOpenEvent
+import at.hannibal2.skyhanni.events.LorenzToolTipEvent
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils.round
+import at.hannibal2.skyhanni.utils.NEUItems
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+import at.hannibal2.skyhanni.utils.StringUtils.matches
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import io.github.moulberry.notenoughupdates.events.ReplaceItemEvent
+import io.github.moulberry.notenoughupdates.util.Utils
+import net.minecraft.client.player.inventory.ContainerLocalMenu
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.seconds
+
+class LimboPlaytime {
+    private lateinit var modifiedArray: MutableList<String>
+    private var setMinutes = false
+    private val minutesRegex by RepoPattern.pattern("limbo.tooltip.minutes", "§5§o§a\\d+(\\.\\d+)? minutes.+\$")
+    private val hoursRegex by RepoPattern.pattern("limbo.tooltip.hours", "§5§o§b\\d+(\\.\\d+)? hours.+\$")
```

Comment:
Just a few things to do with the patterns,
1. Please make the variable name end in pattern not regex
2. Please make these part of a pattern group e.g. "misc.limbo.tooltip"
3. Please split each pattern over 4 lines like every other repo pattern in the code is